### ISSUE TITLE
Wrong RFC resolved

### ISSUE THREAD

**Issue by EyMaddis** [jbenner-radham/rfc-to-bib#2]
Hi,
great tool! I just noticed that the RFC `3986` (URIs) resolved to `66`

https://runkit.com/5931d9902a105b0012742b93/593431619cc3d40012553db8

```
@techreport{rfc66,
    author       = {T. Berners-Lee and R. Fielding and L. Masinter},
    title        = {Uniform Resource Identifier (URI): Generic Syntax},
    howpublished = {Internet Requests for Comments},
    type         = {RFC},
    number       = 66,
    pages        = {1-61},
    year         = 2005,
    month        = January,
    issn         = {2070-1721},
    publisher    = {RFC Editor},
    institution  = {RFC Editor},
    url          = {http://www.rfc-editor.org/rfc/rfc66.txt}
}
```

**Comment by jbenner-radham** [comment]
Hi, thanks for the bug report! I'll take a look at it and see what's going wrong.

**Comment by jbenner-radham** [comment]
Fixed and published in release `0.2.1`.
